Makefile

```diff
@@ -0,0 +1,92 @@
+.DEFAULT_GOAL	:= build
+
+#------------------------------------------------------------------------------
+# Variables
+#------------------------------------------------------------------------------
+
+SHELL		:= /bin/bash
+PKG			:= github.com/disneystreaming/ssm-helpers
```

Comment:
supernitpick.

Do we want to format/align variables and their values?

ex:
```
SHELL		:= /bin/bash
PKG		:= github.com/disneystreaming/ssm-helpers
IMAGE		:= docker.pkg.github.com/disneystreaming/ssm-helpers/ssm
```

Reply:
If you can get this to align better please let me know. Tabs are absolutely horrible.